debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskPartition.java

```diff
@@ -41,6 +41,14 @@ public SqlServerTaskPartition(String serverName, String databaseName) {
         return partition;
     }
 
+    @Override
+    public String toString() {
```

Comment:
The string representation is used here: https://github.com/sugarcrm/debezium/blob/e3159e151d40ca524edf1e0db6a106321dbb061f/debezium-core/src/main/java/io/debezium/pipeline/metrics/PipelineMetrics.java#L85-L89
Also, it could be used for improved logging (a case to be filed).